src/pages/Books.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from '../components/Book';
 import AddBook from '../components/AddBook';
 
-const Books = () => (
-  <div className="container" style={{ padding: '1em 3em' }}>
-    <div className="book-list">
-      <Book title="The Hunger Games" author="Suzanne Collins" id="1" />
+const Books = () => {
+  const books = useSelector((state) => state.books);
+  return (
+    <div className="container">
+      <div className="book-list">
+        { books.map((book) => (
+          <Book key={book.id} title={book.title} author={book.author} id={book.id} />
+        ))}
+      </div>
```

Comment:
- [ ] **[OPTIONAL]:** You did great in this milestone! 💯 However, it could be even better if you change this `<div>` container to an `<ul>` which provides a better semantic value to your page. You're also using an `<li>` tag as a container for each book so at the moment this is not valid HTML. 🤓 🔧